ref_builder/snapshotter/otu.py

```diff
@@ -182,58 +165,55 @@ def _otu_path(self):
         """The path to the OTU's taxonomy data."""
         return self.path / "otu.json"
 
-    @property
-    def _toc_path(self):
-        """The path to the OTU's table of contents."""
-        return self.path / "toc.json"
-
-    @property
-    def _excluded_path(self):
-        """The path to a list of accessions that should not be fetched in the future."""
-        return self.path / "excluded.json"
-
     def cache(
         self,
         otu: "RepoOTU",
         at_event: int | None = None,
-        indent: int | None = None,
     ):
         """Cache an OTU at a given event."""
         self._metadata.at_event = at_event
 
-        try:
-            validated_otu = OTUSnapshotOTU.model_validate(otu.dict(exclude_contents=True))
-        except ValidationError:
-            msg = f"{otu.dict(exclude_contents=True)} is not a valid OTU."
-            raise ValueError(msg)
-
-        with open(self._otu_path, "w") as f:
-            f.write(validated_otu.model_dump_json(indent=indent))
-
-        with open(self._excluded_path, "wb") as f:
-            f.write(orjson.dumps(list(otu.excluded_accessions)))
+        with open(self._otu_path, "wb") as f:
+            f.write(
+                orjson.dumps(
+                    {
+                        "at_event": at_event,
+                        "data": {
+                            "acronym": otu.acronym,
+                            "id": otu.id,
+                            "excluded_accessions": list(otu.excluded_accessions),
+                            "legacy_id": otu.legacy_id,
+                            "name": otu.name,
+                            "repr_isolate": otu.repr_isolate,
+                            "schema": otu.schema.model_dump(),
+                            "taxid": otu.taxid,
+                        },
+                    },
+                ),
+            )
 
         for isolate in otu.isolates:
             self._data.cache_isolate(isolate)
 
             for sequence in isolate.sequences:
                 self._data.cache_sequence(sequence)
 
-        self._toc.write(data=OTUSnapshotToC.generate_from_otu(otu), indent=indent)
+        self._toc.write(data=OTUSnapshotToC.generate_from_otu(otu))
 
-        self._write_metadata(indent=None)
+        with open(self._metadata_path, "w") as f:
+            f.write(self._metadata.model_dump_json())
```

Comment:
I assume this write needs to be removed, now that `at_event` is already included in `otu.json`

Of course, removing `metadata.json` has knock-on effects in `tests/snapshotter/test_otu.py`, so those would need some looking at as well. Specifically:
- [`TestOTUSnapshot.test_snapshot_direct()`](https://github.com/virtool/ref-builder/blob/d490e7313029e7f1d187975089edfc641be59760/tests/snapshotter/test_otu.py#L26) 
- [`test_snapshot_metadata()`](https://github.com/virtool/ref-builder/blob/d490e7313029e7f1d187975089edfc641be59760/tests/snapshotter/test_otu.py#L39)

Reply:
I'll check it out.